### PR TITLE
Mailet 112 CI tests

### PR DESCRIPTION
**Pull request by chibenwa**
CI for https://github.com/apache/james-project/pull/58

### CODE REVIEW

File: mailet/standard/src/test/java/org/apache/james/transport/matchers/FetchedFromTest.java [hunk 1]
```diff
@@ -20,105 +20,70 @@
...
-        setupMockedMimeMessage();
-        setupMockedMail(mockedMimeMessage);
-        setupMatcher();
-
-        Collection<MailAddress> matchedRecipients = matcher.match(mockedMail);
-        Assert.assertNull(matchedRecipients);
+        assertThat(matcher.match(fakeMail)).isNull();
     }
 
     // test if the Header was removed after matched
```
Comment (rouazana) [inline mailet/standard/src/test/java/org/apache/james/transport/matchers/FetchedFromTest.java]: I think you can remove these perfectly redundant comments

**Comment by rouazana** [comment]
:+1:

File: mailet/standard/src/test/java/org/apache/james/transport/matchers/FetchedFromTest.java [hunk 2]
```diff
@@ -20,105 +20,70 @@
...
+        FakeMatcherConfig matcherConfig = new FakeMatcherConfig("FetchedFrom=" + EXPECTED_HEADER_VALUE, FakeMailContext.defaultContext());
+        matcher.init(matcherConfig);
+
+        mailAddress1 = new MailAddress("me@apache.org");
+        mailAddress2 = new MailAddress("you@apache.org");
     }
 
     // test if the Header was matched
     @Test
     public void testHeaderIsMatched() throws MessagingException {
```
Comment (mbaechler) [inline mailet/standard/src/test/java/org/apache/james/transport/matchers/FetchedFromTest.java]: test names are not ok

**Comment by chibenwa** [comment]
Merged. I changed method names before.

**Pull request closed without merging**